pxr/usd/plugin/usdAbc/alembicReader.cpp

```diff
@@ -95,6 +95,12 @@ TF_DEFINE_ENV_SETTING(
     USD_ABC_XFORM_PRIM_COLLAPSE, true,
     "Collapse Xforms containing a single geometry into a single geom Prim in USD");
 
+#if ALEMBIC_LIBRARY_VERSION >= 10709
```

Comment:
how about updating FindAlembic to generate this automatically?

Reply:
What do you mean ?

Reply:
Right now we would need to define `ALEMBIC_LIBRARY_VERSION` manually but the FindAlembic cmake module could have some logic included to set it automatically when it finds the libraries.  Just an idea though.

Reply:
Unless I'm mistaken ALEMBIC_LIBRARY_VERSION is defined in the Alembic headers.

Reply:
Oh, you're right. https://github.com/alembic/alembic/blob/master/lib/Alembic/Util/Config.h.in#L42 perfect :)